cvat-ui/src/reducers/tasks-reducer.ts

```diff
@@ -238,6 +240,52 @@ export default (state: TasksState = defaultState, action: AnyAction): TasksState
                 },
             };
         }
+        case TasksActionTypes.BACKUP_TASK: {
+            const { taskID } = action.payload;
+            const { backups } = state.activities;
+
+            backups[taskID] = true;
+
+            return {
+                ...state,
+                activities: {
+                    ...state.activities,
+                    backups: {
+                        ...backups,
```

Comment:
Try something like this instead of changing current state

``backups: { ...backups, ...Object.fromEntries([[taskID, true]]) })``